src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -446,7 +446,8 @@ class ChatSwitcherView extends React.Component {
                     ));
 
                     // Make sure we don't include the same option twice (automatically handled by using a `Set`)
-                    if (isMatch && !isInGroupUsers) {
+                    // We must also ignore the user if it matches the currently logged in user.
+                    if (isMatch && !isInGroupUsers && this.props.session.email !== option.login) {
```

Comment:
Technically we don't need a new variable for this, but I think it would help readability and follow the existing convention.  So, maybe we can move this to a new line so it's a bit clearer e.g. if this was 

```js
if (isMatch && !isInGroupUsers && !isCurrentlyLoggedInUser) { 
```

it would be very easy to read and we probably don't need that comment at all. Thoughts?

Reply:
Yes of course!  If I'm understanding you correctly, I can add in the check into line 441 along side the other const initializations, as isCurrentlyLoggedInUser, then do exactly as you recommended.